packages/cli/src/credentials/credentials.controller.ee.ts

```diff
@@ -139,7 +144,7 @@ EECredentialsController.put('/:credentialId/share', async (req: CredentialReques
 	const { shareWithIds } = req.body;
 
 	if (!Array.isArray(shareWithIds) || !shareWithIds.every((userId) => typeof userId === 'string')) {
-		return res.status(400).send('Bad Request');
+		return res.status(400).send({ message: 'Bad request' });
```

Comment:
you can change this to use `.json` instead, as `.send` expects a string. then you don't need to add `Content-Type` explicitly.

```suggestion
		return res.status(400).json({ message: 'Bad request' });
```